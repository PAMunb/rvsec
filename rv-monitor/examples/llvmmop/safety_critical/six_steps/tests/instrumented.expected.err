Test 1
Step.
Operation complete.
Step.
Step.
Step.
Step.
Step.
Operation complete.
Step.
Step.
Step.
Step.
Step.
Step.
Step.
too many steps!
Test 2
Step.
Operation complete.
Step.
Step.
Step.
Step.
Step.
Operation complete.
Step.
Step.
Step.
Step.
Step.
Step.
Operation complete.
Step.
Step.
Step.
Step.
Step.
Operation complete.
