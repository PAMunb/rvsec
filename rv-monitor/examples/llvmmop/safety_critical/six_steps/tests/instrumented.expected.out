Test 1
Step 1
Done in 1 step.
Step 1
Step 2
Step 3
Step 4
Step 5
Done in 5 steps.
Step 1
Step 2
Step 3
Step 4
Step 5
Step 6
Test 2
Step 1
Done in 1 step.
Step 1
Step 2
Step 3
Step 4
Step 5
Done in 5 steps.
Step 1
Step 2
Step 3
Step 4
Step 5
Step 6
Done in 6 steps.
Step 1
Step 2
Step 3
Step 4
Step 5
Done in 5 steps.
